edx_solutions_api_integration/courses/views.py

```diff
@@ -1930,6 +1919,108 @@ def get(self, request, course_id):  # pylint: disable=W0613,W0221
         return Response(data, status=status.HTTP_200_OK)
 
 
+class CompletionList(SecureListAPIView):  # pylint: disable=too-many-ancestors
+    """
+    ### The CompletionList allows clients to view user's course module completion entities
+    to monitor a user's progression throughout the duration of a course,
+    - URI: ```/api/courses/{course_id}/completions```
+    - GET: Returns a JSON representation of the course, content and user and timestamps
+    - GET Example:
+        {
+            "count":"1",
+            "num_pages": "1",
+            "previous": null
+            "next": null
+            "results": [
+                {
+                    "id": 2,
+                    "user_id": "3",
+                    "course_id": "32fgdf",
+                    "content_id": "324dfgd",
+                    "stage": "First",
+                    "created": "2014-06-10T13:14:49.878Z",
+                    "modified": "2014-06-10T13:14:49.914Z"
+                }
+            ]
+        }
+
+    Filters can also be applied
+    ```/api/courses/{course_id}/completions/?user_id={user_id}```
+    ```/api/courses/{course_id}/completions/?content_id={content_id}&stage={stage}```
```

Comment:
@jcdyer, the `stage` query string parameter has to be removed since it is not supported after the recent changes.

Reply:
Updated docstring.